test/lib/jdk/test/lib/util/ForceGC.java

```diff
@@ -34,15 +34,18 @@
 public class ForceGC {
     private final CountDownLatch cleanerInvoked = new CountDownLatch(1);
     private final Cleaner cleaner = Cleaner.create();
+    private Object o;
+
     public ForceGC() {
-        cleaner.register(new Object(), () -> cleanerInvoked.countDown());
+        this.o = new Object();
+        cleaner.register(o, () -> cleanerInvoked.countDown());
     }
 
-    private void doit() {
+    private void doit(int iter) {
         try {
             for (int i = 0; i < 10; i++) {
                 System.gc();
-                System.out.println("gc " + i);
+                System.out.format("doit %d: gc %d%n", iter, i);
```

Comment:
Could be better to keep using println(), if performance is important.

Reply:
I agree we don't need format() here